src/TodoList.js

```diff
@@ -0,0 +1,31 @@
+import * as React from 'react';
+
+let todoList = [
+    {
+      id: '1',
+      title: 'Read the first chapter of the book and view videos',
+    },
+    {
+      id: '2',
+      title: 'Complete the assignment',
+    },
+    {
+      id: '3',
+      title: 'Submit the assignment',
+    },
+  ];
+
+function TodoList(){
+    return (
+        <ul>
+        {todoList.map(function (item) {
```

Comment:
lines 21 thru 25 should be indented as it's a new part contained in the list and the curly brackets on line 25 should match line 21.